src/main/java/duke/Event.java

```diff
@@ -12,15 +19,28 @@ public Event(String action, LocalDateTime at) {
         this.at = at;
     }
 
+    /**
+     * Gives the date of the event as output
+     * @return date (in dd-MMM-yyyy)
+     */
     public String getDate() {
```

Comment:
seems that the "getDate" and "getTime" methods are repeated in Deadline.java and Event.java. Might want to consider shifting them to the Task.java class to avoid repetition, or create a new class to handle date and time.